llvm/lib/ObjCopy/ELF/ELFObject.cpp

```diff
@@ -2651,7 +2651,7 @@ Error BinaryWriter::write() {
 
   assert(LoadableSections.front()->Offset == 0);
 
-  for (unsigned i = 0; i != LoadableSections.size(); ++i) {
+  for (std::size_t i = 0; i != LoadableSections.size(); ++i) {
```

Comment:
Nit: `size_t` seems to be the norm rather than `std::size_t` (in my current checkout, there are less than 500 lines referencing `std::size_t` versus over 12500 referencing `size_t` (so presumably ~12000 that actually use `size_t` and not `std::size_t`).

Reply:
Done.